### DISCUSSION TITLE
How to install wordpress in subdirectory / subfolder of existing installation?

### DISCUSSION THREAD

**Discussion by KoolPal** [leokhoa/laragon#677] [Q&A]
Hi All,

Is there a guide on how to install wordpress in subdirectory / subfolder of existing installation in Laragon?

I am NOT looking at a sub domain instance.

I am trying to achieve
    Main site: yoursite.com/
    Second instance: yoursite.com/shop2/

Thanks

**Comment by dustinatx** [comment]
I'm pretty sure it would be the same as installing WordPress normally. As in, using the instructions on WordPress.org, not using Quick app.

https://developer.wordpress.org/advanced-administration/before-install/howto-install/

**Comment by KoolPal** [comment]
Thanks @dustinatx 

I realized that Quick App would not work, hence asked.

Any idea on what other manual changes would be required (if any) within Laragon settings to make this site work locally?

Thanks

**Comment by dustinatx** [comment]
Nothing comes to mind. I've never installed WordPress in this way in Laragon.

**Comment by ghost** [comment]
this discussion is old, but perhaps it will be help:

ty replacing the following in the site.conf in wordpress

# WordPress
WordPress=https://wordpress.org/latest.tar.gz
# Automatically installs WordPress in a subfolder named 'wordpress'
WordPress-folder=%s/wordpress

I've manupulate my site.conf very hard with some own entries for other cms and subfoders. 
For me on win 11 it works
